cclient/iota_api.c

```diff
@@ -0,0 +1,101 @@
+// Copyright 2018 IOTA Foundation
+
+#include "iota_api.h"
+
+iota_api_result_t iota_api_get_node_info(const iota_service_t* const service,
+                                         get_node_info_res_t* const res) {
+  iota_api_result_t result;
```

Comment:
Since you are about to implement a set of dummy functions as hooks, how about prepare a dedicated macro like `DUMMY_FUNC_BODY` which generates function body?

Reply:
Sure, but then i'll have to apply it on every function separately
so it could have been nice to do that beforehand